kratos/geometries/geometry.h

```diff
@@ -528,8 +528,15 @@ class Geometry : public PointerVector<TPointType>
      * @see Volume()
      */
     virtual double DomainSize() const {
-      KRATOS_ERROR << "Calling base class 'DomainSize' method instead of derived class one. Please check the definition of derived class. " << *this << std::endl;
-      return 0.0;
+        const SizeType local_space_dimension = this->LocalSpaceDimension();
```

Comment:
Additionally to what is defined in #3729 I added this generic version of DomainSIze